va/va_vpp.h

```diff
@@ -336,6 +406,21 @@ typedef enum _VAProcColorStandardType {
 #define VA_PIPELINE_FLAG_END		0x00000004
 /**@}*/
 
+/** @name Chroma Siting flag */
+/**@{*/
+#define VA_CHROMA_SITING_UNKNOWN              0x00000000
```

Comment:
This presumably actually means `VERTICAL_CENTRE | HORIZONTAL_LEFT` (i.e. the default for most video codecs) in all real cases?  Maybe just say that.

Reply:
In driver, if application does not specify chromasiting information, driver will use left/center as default chromasiting.